tests/request_helpers_test.py

```diff
@@ -0,0 +1,408 @@
+# -*- coding: utf-8 -*-
+import mock
+import pytest
+
+import py_zipkin.request_helpers as request_helpers
```

Comment:
why not `from py_zipkin import request_helpers`?

Reply:
Whoops, just a brain fart; fixing